mir/dlsite/api.py

```diff
@@ -121,6 +122,17 @@ def _get_description(soup) -> str:
     return text.strip() + '\n'
 
 
+def _get_images(soup) -> 'List[str]':
+    return list(_generate_images(soup))
+
+
+def _generate_images(soup) -> 'Iterable[str]':
+    div = soup.find('div', {'class': 'product-slider-data'})
+    image_div = div.find_all('div')
```

Comment:
It appears all works are required to have at least one image so this should never fail.